src/controllers/dynakube/oneagent/daemonset/daemonset_test.go

```diff
@@ -500,85 +500,76 @@ func TestPodSpecServiceAccountName(t *testing.T) {
 	})
 }
 
-func TestPodSpecReadinessProbe(t *testing.T) {
-	testCommands := []string{"echo", "super pod"}
-	interval := time.Second * 10
-	timeout := time.Second * 30
-	startPeriod := time.Second * 1200
-	retries := 3
-
-	t.Run("set readiness probe when dynakube oneagent status has healthcheck", func(t *testing.T) {
-		builder := builderInfo{
-			dynakube: &dynatracev1beta1.DynaKube{
-				Status: dynatracev1beta1.DynaKubeStatus{
-					OneAgent: dynatracev1beta1.OneAgentStatus{
-						Healthcheck: &containerv1.HealthConfig{
-							Test:        testCommands,
-							Interval:    interval,
-							Timeout:     timeout,
-							StartPeriod: startPeriod,
-							Retries:     retries,
-						},
-					},
-				},
-			},
-		}
-		podSpec := builder.podSpec()
+func TestPodSpecProbes(t *testing.T) {
+	expectedHealthcheck := containerv1.HealthConfig{
+		Test:        []string{"echo", "super pod"},
+		Interval:    time.Second * 10,
```

Comment:
yeah I know it was there before you changed, but anyway better to use `<NUM> * time.Second` style, it's more readable IMO.

Example https://pkg.go.dev/time#example-After

cc @0sewa0 wdyt if we add it to our guide?

Reply:
wasn't able to find an existing rule, most similar seems to be for [order of operands in ifs](https://github.com/mgechev/revive/blob/master/RULES_DESCRIPTIONS.md#optimize-operands-order)